### PR TITLE
Hello Microverse!

### PR DESCRIPTION
**Pull request by sergio-usma**
This project reviewed some functionalities of linters, GIT, and GitHub, fulfilling the requirements of the activity:
- Created a new GitHub repo.
- Created a new branch according to the GitHub flow rules.
- Added a .gitignore file.
- Added node_modules to the gitignore file.
- Set up a linter for HTML and CSS making sure that any files that were not relevant were ignored by git.
- Added a descriptive README file to the project using a customized template.
- Added 3 files (each one in a separate commit):
- An HTML file that included a header with the text "Hello Microverse!" using h1 tag.
- A CSS file that included one class with styles for the header in my favorite color.
- A test.md file with the text "This file should be ignored by git".
- Opened a pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,237 @@
...
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./murple_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (Alfredbis29) [inline README.md:30]: - [ ] Great work so far for this project , kindly remove the template logo and use your own logo for your project to look more professional :wink:

File: README.md [hunk 2]
```diff
@@ -1,2 +1,237 @@
...
+### Prerequisites
+
+In order to run this project you need:
+
+- VS Code or an IDE
+- NodeJS
+- Git
+- GitHub account
+
+### Setup
```
Comment (Alfredbis29) [inline README.md:111]: - [ ] I can see that you tried to give the way on how we can setup the project in our local machine but you did provide the Project Link which we are going to clone , kindly add it for being more specific on how to setup the project :smile:

File: README.md [hunk 3]
```diff
@@ -1,2 +1,237 @@
...
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
```
Comment (Alfredbis29) [inline README.md:122]: - [Optional ]Kindly remove unused comments in your Readme and customize it on your own :wink:

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
#  STATUS: REQUIRED CHANGES 
 ## Hi @sergio-usma ,
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  
 ## Highlights 
 :heavy_check_mark: Linters are passing.
:heavy_check_mark: You have a good code structure.
:heavy_check_mark: Your linters are correct.
 
 ## Required Changes 
_Check Under the Review

If you need any suggestion from me I suggest you to use tabindex in your headings paragraph tags to make text . :upside_down_face:
 
 ## Optional suggestions 

- [ ]Check_under the review 

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sergio-usma) [reply README.md]: Updated!

File: README.md (hunk 2 above)
Comment (sergio-usma) [reply README.md:105]: Updated!

File: README.md (hunk 3 above)
Comment (sergio-usma) [reply README.md]: Updated! Thank you very much for the feedback!

**Review by divinecharlotte — APPROVED** [review]
# Hello @sergio-usma    😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- No linters errors ✔️
- Descriptive PR ✔️
- Descriptive commit messages 👍 





## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**